project-generated-code-backend/Backend/Model/Accounts/Patient.cs

```diff
@@ -26,8 +25,6 @@ public Patient(string name, string surname, string id, DateTime dateOfBirth, str
             this.Guest = isGuest;
             this.password = password;
         }
-
-        [JsonConstructor]
         public Patient(string serialNumber, string name, string surname, string id, DateTime dateOfBirth, string contact, string email, Address address, string parentName, string gender, string password, bool isGuest = false)
             : base(serialNumber, name, surname, id, dateOfBirth, contact, email, address)
         {
```

Comment:
The [JsonConstructor] annotation is necessary here or the code that uses FileSystem will crash, I think you've deleted that by accident.